pyomo/solvers/plugins/solvers/GAMS.py

```diff
@@ -722,7 +755,8 @@ def solve(self, *args, **kwds):
         ####################################################################
 
         exe = self.executable()
-        command = [exe, output, "o=" + lst, "curdir=" + tmpdir]
+        command = [exe, output, "o=" + lst, "curdir=" + tmpdir, "solvelink=5", 
+                   "limrow=0", "limcol=0", "solprint=off"]
```

Comment:
`solvelink`, `limrow`, `limcol` and `solprint` can be set via options. Do we really want to enforce them by default here for all users?

Reply:
Very good point. I haven't thought about that earlier. I would suggest to use those values on default, but if a user wishes to use something else, that should be fine. This may also apply to the `$offlisting` in https://github.com/Pyomo/pyomo/pull/1446/files#diff-44a99086316025ea73571ead9ddbfb19R581.

Reply:
`$offlisting` I'm not too worried about, since if `keepfiles=True`, they have the original `*.gms` file right next to it.

I should clarify: I am not opposed to having these be the default options, but I worry that feeding them in via command line here might override options set within `add_options`.

Reply:
I agree, I think that overwriting any options that the user thinks they are setting could be potentially problematic.

Reply:
These options are now set through gams option statements in the gams_writer if they haven't been set otherwise.